### DISCUSSION TITLE
"Bug" Unable to play video, "Quality 0p-limited"

### DISCUSSION THREAD

**Discussion by deception-andpitfalls** [libre-tube/LibreTube#5280] [Q&A]
I started encountering this recently, the "Quality" placeholder is always constant on "0p- limited" & can't be changed.

![Screenshot_20231203-181316.png](https://github.com/libre-tube/LibreTube/assets/105056343/c15e725a-d707-436a-941d-b295c77d6309)

Everything looks normal but unable to play the video

![Screenshot_20231203-181334.png](https://github.com/libre-tube/LibreTube/assets/105056343/5978e7e0-fb30-488a-9d2c-8a887e05d77a)
![Screenshot_20231203-182203.png](https://github.com/libre-tube/LibreTube/assets/105056343/0f77c3ca-68fa-4e6e-95d3-4bf24dfaf183)

**Comment by Profi420** [comment]
Same here

**Reply by deception-andpitfalls** [reply]
It seems to be working fine today!

![1000018640-01.jpeg](https://github.com/libre-tube/LibreTube/assets/105056343/931e5a64-918a-4c56-820f-ed6cdd1f6487)

I haven't used “ReVanced Extended” for a while for similar reasons, I updated it along with “MicroG” yesterday, nothing changed but after changing the option “Spoof player parameter” everything functions.

Not that it has anything to do with the app, but do keep backup up applications in case of nefarious manipulations.

![1000018642-01.jpeg](https://github.com/libre-tube/LibreTube/assets/105056343/8fee8cd4-6fb0-447a-ab52-c64be8a43080)

**Comment by MinekPo1** [comment]
I experience this issue now and again, usually switching instances gets rid of the issue. Likely some connection issue?
